src/main/java/cms/gov/madie/measure/exceptions/SpecialCharacterException.java

```diff
@@ -0,0 +1,11 @@
+package cms.gov.madie.measure.exceptions;
+
+public class SpecialCharacterException extends RuntimeException {
+  private static final long serialVersionUID = 4113761000508937686L;
+
+  private static final String MESSAGE = "Test Case %s can not contain special characters";
```

Comment:
same here, I think we should let the caller know the restricted character set

Reply:
@adongare Changed too.

Reply:
@adongare Will do thanks Ashok!

Reply:
@adongare this is done, thanks!